src/ansys/fluent/core/solver/flobject.py

```diff
@@ -243,6 +244,33 @@ def set_state(self, state: StateT = None, **kwargs):
         else:
             return self.flproxy.set_var(self.path, self.to_scheme_keys(state))
 
+    def find_child(self, identifier="*"):
+        """Lists path of all the child objects matching an identifier."""
+        _list_of_children = []
```

Comment:
Local variable need not/should not be prefixed with _.

Reply:
Done, thank you